tests/Tokenizer/Transformer/NullableTypeTransformerTest.php

```diff
@@ -135,4 +135,39 @@ public function provideProcess74Cases()
             ],
         ];
     }
+
+    /**
+     * @param string $source
+     *
+     * @dataProvider provideProcess80Cases
+     * @requires PHP 8.0
+     */
+    public function testProcess80($source, array $expectedTokens = [])
+    {
+        $this->doTest(
+            $source,
+            $expectedTokens,
+            [
+                CT::T_NULLABLE_TYPE,
+            ]
+        );
+    }
+
+    public function provideProcess80Cases()
+    {
+        return [
+            [
+                '<?php
+                    class Foo
+                    {
+                        public function __construct(private ?string $foo = null)
```

Comment:
as we have explicit list of private/protected/public in source file, can we add 3 variants here, please?

Reply:
@keradus Good point :+1: Fixed